# ManipulationTask.action

# ---- Request (Goal) ----
string target_station

---
# ---- Result ----
bool success
int32 error_code
string error_msg

---
# ---- Feedback ----
float32 progress